rsocket-core/src/main/java/io/rsocket/RSocketFactory.java

```diff
@@ -499,7 +498,6 @@ public ServerRSocketFactory fragment(int mtu) {
 
     /** @deprecated this is deprecated with no replacement. */
     public ServerRSocketFactory errorConsumer(Consumer<Throwable> errorConsumer) {
-      server.errorConsumer(errorConsumer);
       return this;
     }
```

Comment:
Same here, mention the method is a no-op.

Reply:
fixed